javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -2,7 +2,8 @@
 function addSuffixToAll(words, suffix) {
   var suffixArray = [];
   for (var i = 0; i < words.length; i++) {
-    suffixArray[i] = (words[i] += suffix);
+    suffixArray.push(words[i]);
+    suffixArray[i] = (suffixArray[i] += suffix);
```

Comment:
This can be simplified a bit so you only need one line and it doesn't change your words array.

```js
suffixArray.push(words[i] + suffix);
```